contrib/redhat/keyszer.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=keyszer
+
+[Service]
+Type=simple
+KillMode=process
+ExecStart=/usr/bin/keyszer --config /etc/keyszer.conf
```

Comment:
Does RedHat make no distinction between /usr/bin and /usr/sbin?  Shouldn't this be in sbin as as system service or is that only for root stuff?

Reply:
`/usr/bin` and /usr/sbin/` are different folders in RH-based.
'keyszer` is installing into `/usr/bin/` by its installer so I just point to them.

Reply:
Oh, then that's a python thing, ok... that's fine for now then.